.azure-pipelines/ultimate-pipeline.yml

```
@@ -3211,18 +3218,7 @@ stages:
       - publish: system-tests/logs
         condition: always()
         displayName: System tests logs
-        artifact: _$(System.StageName)_$(Agent.JobName)_logs_$(System.JobAttempt)
-
-      - script: ./run.sh UDS
-        workingDirectory: system-tests
-        displayName: Run tests over UDS
-        env:
-          DD_API_KEY: $(SYSTEM_TESTS_DD_API_KEY)
-        
-      - publish: system-tests/logs_uds
-        condition: always()
-        displayName: System tests UDS logs
-        artifact: _$(System.StageName)_$(Agent.JobName)_logs_uds_$(System.JobAttempt)
+        artifact: _$(System.StageName)_$(Agent.JobName)_logs_$(System.JobAttempt)_$WEBLOG_VARIANT
```

This change shouldn't be necessary as the `JobName` includes the WEBLOG_VARIANT

```suggestion
        artifact: _$(System.StageName)_$(Agent.JobName)_logs_$(System.JobAttempt)
```